### DISCUSSION TITLE
buddhism/upside_down/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#54] [General]
# buddhism/upside_down/

https://shanenull.com/buddhism/upside_down/

**Comment by shane0** [comment]
https://youtu.be/2RavAQoqxOY

**Comment by shane0** [comment]
According to Buddhist teachings, the three root causes of suffering are:

1. **Ignorance (Avidya):** This refers to a lack of understanding of the true nature of reality. Ignorance leads to us clinging to the illusion of a permanent, unchanging self, which is the source of much suffering.

2. **Attachment (Tanha):** This refers to the craving for things that are ultimately impermanent and unsatisfying. Attachment leads to us constantly chasing after things that cannot truly satisfy us, which results in frustration and disappointment.

3. **Aversion (Dvesha):** This refers to the aversion to things that we dislike or fear. Aversion leads to us pushing away from things that we don't want to experience, which creates tension and suffering.

The Buddha taught that the only way to escape from suffering is to let go of ignorance, attachment, and aversion. By cultivating wisdom, compassion, and acceptance, we can begin to see the world as it truly is and find peace and liberation from suffering.
